spicy/runtime/src/driver.cc

```diff
@@ -232,6 +240,11 @@ driver::ParsingState::State driver::ParsingState::_process(size_t size, const ch
                 auto input = hilti::rt::reference::make_value<hilti::rt::Stream>(data, size);
                 input->freeze();
 
+                if ( ! _parser->parse1 )
+                    throw InvalidUnitType(
```

Comment:
Since this is an internal function I am not sure where exactly we should document that this exception can be thrown, but we should document it _somewhere_ since this our external API or very close to it.